### PR TITLE
add fields to meta and add key to wallet nodes

### PR DESCRIPTION
**Pull request by g-a-v-i-n**
- add patp, tier and derivationPath to `meta` property in output wallet
- gitignored dist
- add reference keys in each wallet node. Useful for downstream tests.

I still have to update one test for all tests to pass, but I want to ask for review before I do this because updating the test is tedious. I made these changes to reduce complexity in PaperCollateralRenderer.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -27206,41 +27206,46 @@ utils.intFromLE = intFromLE;
 
 },{"bn.js":19,"minimalistic-assert":77,"minimalistic-crypto-utils":78}],48:[function(require,module,exports){
 module.exports={
-  "_from": "elliptic@^6.2.3",
+  "_args": [
+    [
+      "elliptic@6.4.1",
+      "/Users/gavinatkinson/Tlon/urbit-key-generation"
```
Comment (Fang-) [inline dist/index.js]: I was gonna ignore `dist` contents, but these local paths don't seem quite right...

Comment (g-a-v-i-n) [reply dist/index.js]: I think we should gitignore /dist. Can do so in another PR.

Comment (g-a-v-i-n) [reply dist/index.js]: I didn't realize generate_test_wallets.js did basically the same thing. I can just remove the /tools dir in that case.

Comment (Fang-) [reply dist/index.js]: Agreed about gitignore!

Comment (g-a-v-i-n) [reply dist/index.js]: I have gitignored dist

File: src/index.js [hunk 2]
```diff
@@ -368,11 +373,13 @@ const generateOwnershipWallet = async config => {
...
   const masterSeed = await argon2u(buf, ship)
 
-  return deriveNode(
+  const node = deriveNode(
     masterSeed,
     CHILD_SEED_TYPES.OWNERSHIP,
     passphrase
   );
+
+  return { type: CHILD_SEED_TYPES.OWNERSHIP, ...node }
```
Comment (Fang-) [inline src/index.js]: Shouldn't the `type` just be returned from `deriveNode` instead?

Comment (g-a-v-i-n) [reply src/index.js]: yes, good catch.

**Pull request merged**